packages/devtools/lib/src/profiler/cpu_profile_processor.dart

```diff
@@ -6,9 +6,9 @@ import 'package:meta/meta.dart';
 import '../utils.dart';
 import 'cpu_profile_model.dart';
 
-/// Protocol for processing [CpuProfileData] and composing it into a structured
-/// tree of [CpuStackFrame]'s.
-class CpuProfileProtocol {
+/// Process for composing [CpuProfileData] into a structured tree of
```

Comment:
Nit: I know I suggested Processor in the first place but I have a better idea. How about
CpuProfileTransformer.
Then this comment would read better.

Reply:
done.